README.md

```diff
@@ -1 +1,105 @@
-# angular-http-interceptors-readme
\ No newline at end of file
+# HTTP Interceptors
+
+## Overview
+
+## Objectives
+
+- Describe HTTP Interceptors
+- Write an HTTP Interceptor
+
+## What is a HTTP interceptor?
+
+Now, using `$http` or `$resource` means we're going to end up doing a lot of HTTP requests in our applications.
+
+Eventually, we may need to manipulate all of the requests, or even take a wide-look at all the responses and handle any errors globally instead of on an individual case - we can do this with HTTP interceptors.
```

Comment:
Can you add a couple of real world examples here of things you may want to do with an interceptor? How might you handle getting an error on a request to create a new user?